### PR TITLE
[CI] Update frame rate row in xcmetrics report

### PR DESCRIPTION
**Pull request by testableapple**
### 📝 Summary

In our xcmetrics report we assumed that the benchmark should always be lower than the branch value. This is wrong for the `frame_rate` metric. It turned out that the higher fps, the better 🙂

### ☑️ Contributor Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] This change follows zero ⚠️ policy (required)
- [ ] This change should be manually QAed
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -400,23 +400,38 @@ lane :xcmetrics do |options|
...
+            fail_status
+          else
+            success_status
+          end
         end
 
       benchmark_value_avoids_zero_division = benchmark_value == 0 ? 1 : benchmark_value
       diff = ((benchmark_value - branch_value) * 100.0 / benchmark_value_avoids_zero_division).round(2)
-      diff_emoji = diff > 0 ? '🔼' : '🔽'
+      diff_emoji = (is_frame_rate && diff < 0) || (!is_frame_rate && diff > 0) ? '🔼' : '🔽'
```
Comment (laevandus) [inline fastlane/Fastfile:434]: Can we add equal sign here as well when there is no difference

**Review by laevandus — APPROVED** [review]

File: fastlane/Fastfile (hunk 1 above)
Comment (testableapple) [reply fastlane/Fastfile:434]: Then share the emoji you'd like to see in this case 😄

Comment (laevandus) [reply fastlane/Fastfile:434]: 🟰:)

**Review by nuno-vieira — APPROVED** [review]

File: fastlane/Fastfile (hunk 1 above)
Comment (testableapple) [reply fastlane/Fastfile]: done

**Pull request merged**